build/webpack.config.js

```diff
@@ -58,6 +58,7 @@ const languages = isRelease
 function updateConfig(conf, language, index) {
     const config = Object.assign(conf, {
         entry: {
+            boxAnnotations: ['box-annotations/lib/BoxAnnotations'],
```

Comment:
Thoughts on renaming this to annotations? So that the file loads as annotations.js

Reply:
Sure, we're going to have to test your change to see about annotations.js vs. annotations.min.js